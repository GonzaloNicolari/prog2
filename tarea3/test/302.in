# prueba de izquierdo, derecho, altura_binario, imprimir_binario, buscar_subarbol
altura_binario
imprimir_binario
insertar_en_binario (8,r)
altura_binario
imprimir_binario
izquierdo
derecho
insertar_en_binario (6,y)
altura_binario
imprimir_binario
izquierdo
derecho
insertar_en_binario (6,t)
altura_binario
insertar_en_binario (4,m)
altura_binario
imprimir_binario
izquierdo
derecho
insertar_en_binario (4,g)
insertar_en_binario (4,d)
insertar_en_binario (4,h)
insertar_en_binario (4,i)
insertar_en_binario (4,j)
altura_binario
imprimir_binario
izquierdo
derecho
Fin



