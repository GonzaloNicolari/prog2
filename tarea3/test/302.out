1>#  prueba de izquierdo, derecho, altura_binario, imprimir_binario, buscar_subarbol.
2>0.
3>
4>Insertado.
5>1.
6>
(8,r)
7>
8>
9>Insertado.
10>2.
11>
-(6,y)
(8,r)
12>
13>
(6,y)
14>Insertado.
15>3.
16>Insertado.
17>3.
18>
-(6,y)
--(6,t)
(8,r)
-(4,m)
19>
(4,m)
20>
(6,y)
-(6,t)
21>Insertado.
22>Insertado.
23>Insertado.
24>Insertado.
25>Insertado.
26>6.
27>
-(6,y)
--(6,t)
(8,r)
-(4,m)
-----(4,j)
----(4,i)
---(4,h)
--(4,g)
---(4,d)
28>
(4,m)
----(4,j)
---(4,i)
--(4,h)
-(4,g)
--(4,d)
29>
(6,y)
-(6,t)
30>Fin.
